holmes/core/conversations_worker/realtime_manager.py

```
@@ -0,0 +1,705 @@
+                    max(0.01, next_refresh_at - asyncio.get_running_loop().time()),
+                )
+                await asyncio.sleep(sleep_for)
+        except Exception:
+            logging.exception("Error in realtime manager main loop", exc_info=True)
+        finally:
+            self._connected = False
+            # Wake the worker so it falls back to polling
+            try:
+                self.on_new_pending()
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)